### ISSUE TITLE
Accelerometer Support for Mobile.

### ISSUE THREAD

**Issue by blackman91** [libretro/RetroArch#4374]
Could you add the ability to use tilt input in retroarch? It would be cool to play classic racing games like crash team racing and mario kart using tilt. Almost all android emulators support this feature, it would be awesome for retroarch to support tilt input for all cores in the foreseeable feature, it would work great for tablets and phones.

**Comment by meepingsnesroms** [comment]
Do you mean for the analog sticks?

Because SNES and GBA Mario kart would play very poorly with that since there would be a huge dead zone then turning as fast as possible.

**Comment by mafrailea** [comment]
Exactly, I would love to play The Need For Speed SE with gyro controls instead of having to use the chunky touch controls.

Yes, I could use a bluetooth gamepad instead, but gyro controls would feel much more natural for racing games.

**Comment by RubyRiolu** [comment]
This would also be very useful for Kirby Tilt ‘n’ Tumble
